daal4py/sklearn/_utils.py

```diff
@@ -257,3 +266,119 @@ def get_status(self, logs=False):
         if logs:
             self.write_log()
         return self.patching_is_enabled
+
+
+# decorator for addition of "n_jobs" parameter to estimator's init
+def support_init_with_n_jobs(init_function):
```

Comment:
I generally prefer function descriptions to be provided like this. I would argue it's more Pythonic and more widely used. Do we have a guideline on this? We should have one ...

```suggestion
def support_init_with_n_jobs(init_function):
    """decorator for addition of "n_jobs" parameter to estimator's init"""
```

Reply:
Done.
No guidelines exist for this currently.